src/main/java/bio/terra/service/dataset/flight/datadelete/DatasetDataDeleteFlight.java

```diff
@@ -39,6 +39,8 @@ public DatasetDataDeleteFlight(FlightMap inputParameters, Object applicationCont
 
         RetryRuleRandomBackoff lockDatasetRetry =
             new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
+        RetryRuleRandomBackoff unlockDatasetRetry =
```

Comment:
Should this be configurable for locking and unlocking? Or just reuse a single `datasetRetry` rule?

Reply:
I think you're right - I can just set one.